the-blue-alliance-ios/View Controllers/Teams/TeamsViewController.swift

```diff
@@ -160,11 +160,11 @@ extension TeamsViewController: Refreshable {
 
             self.persistentContainer.performBackgroundTask({ (backgroundContext) in
                 // TODO: Delete old teams for page? Kinda a problem but
```

Comment:
This needs an insertPage method that deletes old teams - I've thought out how to do this well enough. Seems fine to add, since we've audited Team at this point.